packages/components/src/components/carousel.tsx

```diff
@@ -128,14 +128,7 @@ class Carousel extends Component<Props, State> {
         const containerCss = cx(className, carouselCss)
         const gifCss = cx(carouselItemCss, marginCss)
         return (
-            <PingbackContextManager
-                attributes={[
-                    {
-                        key: 'layout_type',
```

Comment:
Attributes is a plain object, instead of an array of objects with `key` and `value`,  makes it much easier